languages/auto-tags-smart-uk.po

```diff
@@ -0,0 +1,142 @@
+
+#~ msgid "Auto Tags Smart"
+#~ msgstr "Розумні Авто-позначки"
+
+#~ msgid "Settings"
+#~ msgstr "Налаштування"
+
+#: auto-tags-smart.php:29
+msgid "Select All"
+msgstr "Вибрати все"
+
+#: auto-tags-smart.php:30
+msgid "Deselect All"
+msgstr "Зняти виділення"
+
+#: auto-tags-smart.php:31
+msgid "STATUS: Active ✓"
+msgstr "СТАТУС: Активно ✓"
+
+#: auto-tags-smart.php:32
+msgid "STATUS: Active but nothing to examine"
+msgstr "СТАТУС: Активно, немає доступних записів для аналізу"
+
+#: auto-tags-smart.php:33
+msgid "STATUS: Inactive"
+msgstr "СТАТУС: Неактивно"
+
+#: admin/auto-tags-smart-admin.php:41
+msgid "Enable Auto Tags Smart"
+msgstr "Увімкнути розумні авто-позначки"
+
+#: admin/auto-tags-smart-admin.php:42
+msgid "Enables automatic detection of existing tags in posts."
+msgstr "Увімкнути автоматичне визначення існуючих позначкiв у записах."
+
+#: admin/auto-tags-smart-admin.php:49
+msgid "Block manually added tags"
+msgstr "Видаляти користувацькі позначки"
+
+#: admin/auto-tags-smart-admin.php:50
+msgid "Removes manually added tags and uses only automatically detected ones."
+msgstr "Коли увімкнено, опція видаляє вручну додані позначки та використовує лише автоматично визначені."
+
+#: admin/auto-tags-smart-admin.php:49
+msgid "Examine post title"
+msgstr "Аналізувати заголовки записів"
+
+#: admin/auto-tags-smart-admin.php:50
+msgid "Analyzes the post title to find tags."
+msgstr "Шукати позначки у заголовках записів."
+
+#: admin/auto-tags-smart-admin.php:55
+msgid "Examine post content"
+msgstr "Аналізувати вміст записів"
+
+#: admin/auto-tags-smart-admin.php:56
+msgid "Analyzes the post content to find tags."
+msgstr "Шукати позначки у тексті записів."
+
+#: admin/auto-tags-smart-admin.php:62
+msgid "Filter by category"
+msgstr "Фільтрувати за категоріями"
+
+#: admin/auto-tags-smart-admin.php:63
+msgid "Only applies tags to posts in selected categories."
+msgstr "Застосовувати позначки лише до вибраних категорій."
+
+#: admin/auto-tags-smart-admin.php:68
+msgid "Tag Length"
+msgstr "Довжина позначкiв"
+
+#: admin/auto-tags-smart-admin.php:73
+msgid "Minimum tag length"
+msgstr "Мінімальна довжина позначкiв:"
+
+#: admin/auto-tags-smart-admin.php:74
+msgid "Tags with fewer characters than this will be ignored."
+msgstr "Позначки з меншою кількістю символів будуть ігноруватися."
+
+#: admin/auto-tags-smart-admin.php:79
+msgid "Case sensitive"
+msgstr "Чутливість до регістру"
+
+#: admin/auto-tags-smart-admin.php:80
+msgid "Whether tag matching should be case sensitive."
+msgstr "Коли увімкнено, буде враховуватися регістр при пошуку позначок."
+
+#: admin/auto-tags-smart-admin.php:99
+msgid "Included Categories"
+msgstr "Відстежувані категорії"
+
+#: admin/auto-tags-smart-admin.php:102
+msgid "Clean Uninstall"
+msgstr "Налаштування видалення"
+
+#: admin/auto-tags-smart-admin.php:108
+msgid "Remove all plugin options from the database when uninstalling"
+msgstr "Видаляти всі налаштування з бази даних при видаленні плагіна"

```

Comment:
I see you have changed English label for this option. However, looks like you've forgotten to update it somewhere in the code, since the `msgid` is no longer valid, and the option is being shown in English

<img width="513" height="178" alt="Image" src="https://github.com/user-attachments/assets/56c401f7-8745-4ed2-a17f-a66588110dc0" />

Same is valid for Russian and Portuguese locales.